README.md

```diff
@@ -116,9 +118,13 @@ Failing to do so will produce inconsistencies with the expected results.
 
 From the `its/ruling` folder, launch the ruling tests:
 
-    mvn clean install -DskipTests=false
+    mvn clean install -DskipTests=false -DcommunityEditionTestsOnly=true 
```

Comment:
Here we should replace `-DskipTests=false` with `-Pit-ruling`, because it is the way `.cirrus.yml` execute its `ruling_task:`